.travis.yml

```diff
@@ -5,14 +5,14 @@ services:
 
 env:
   global:
-    - INSTALL_EDM_VERSION=2.4.1
+    - INSTALL_EDM_VERSION=2.0.0
```

Comment:
We probably want to upgrade the version anyway?

Reply:
Yep, but separate PR.